apps/site/sentry.client.config.ts

```diff
@@ -1,7 +1,7 @@
 // Context: https://docs.sentry.io/platforms/javascript/guides/nextjs/
 
+import { Replay } from "@sentry/browser";
```

Comment:
TODO: wait for resolution of https://github.com/getsentry/sentry-javascript/issues/6574